index.js

```diff
@@ -16,4 +16,30 @@
  * limitations under the License.
  */
 
-require('./lib/cli.js');
+const { spawnSync } = require('child_process');
+const fs = require('fs');
+
+let playwrightPackage = 'playwright';
+try {
+  require.resolve('playwright');
+} catch (e) {
+  playwrightPackage = 'playwright-core';
+}
+
+function run(args) {
+  const packageJson = require(playwrightPackage + '/package.json');
+  const cliPath = packageJson.bin.playwright;
+  const cliFile = require.resolve(playwrightPackage + '/' + cliPath);
+  spawnSync(process.argv[0], [cliFile, ...args], { stdio: 'inherit' });
```

Comment:
I maybe miss the background, but why not doing something like: `require('playwright/lib/cli/cli.js')` instead of spawning a sub process?